DatadogCore/Sources/Core/Storage/EventGenerator.swift

```diff
@@ -9,10 +9,10 @@ import DatadogInternal
 
 /// Event generator that generates events from the given data blocks.
 internal struct EventGenerator: Sequence, IteratorProtocol {
-    private let dataBlocks: [DataBlock]
+    private let dataBlocks: [BatchDataBlock]
```

Comment:
we potentially could rename the variable in cases like this for consistency

Reply:
It's `private` API, I'll keep it as it is.